CommonTools/TriggerUtils/test/GenericTriggerEventFlagTest.cc

```diff
@@ -15,11 +15,10 @@
   \version  $Id: GenericTriggerEventFlagTest.cc,v 1.2 2012/01/19 20:17:34 vadler Exp $
 */
 
-#include "FWCore/Framework/interface/EDFilter.h"
-
+#include "FWCore/Framework/interface/stream/EDFilter.h"
 #include "CommonTools/TriggerUtils/interface/GenericTriggerEventFlag.h"
 
-class GenericTriggerEventFlagTest : public edm::EDFilter {
+class GenericTriggerEventFlagTest : public edm::stream::EDFilter<> {
   GenericTriggerEventFlag* genericTriggerEventFlag_;
```

Comment:
Naive question: `genericTriggerEventFlag_` is not const and it looks like it gets modified at Run transitions (`beginRun`). Would this be problematic once this is turned into a stream module?

Reply:
I think that every time a different run is encountered in the stream a run transition with `beginRun` is started. In that sense, this looks thread safe to me for a `stream` module (while it wouldn't be for a `global` one).
But maybe @makortel can canfirm

Reply:
done in the last push.